### ISSUE TITLE
[Feature Request] Streaming recognition

### ISSUE THREAD

**Issue by TemptingBadger** [z411/trackma#395]
Is it possible to implement streaming recognition, like Crunchyroll?
Or, if there is other way to make the same?
Thanks.

**Comment by z411** [comment]
Yeah, I could implement a window name scanning tracker and that should do the job. But I would need the help of contributors to add support for the websites they use.

**Comment by TemptingBadger** [comment]
Ok, I haven't a knowledge enough in programming but I could help if you want.

**Comment by z411** [comment]
Yes, it would be helpful if you told me what browsers and streaming websites you use.
